vyper/semantics/validation/module.py

```diff
@@ -168,7 +171,7 @@ def visit_AnnAssign(self, node):
 
                     # generate function type and add to metadata
                     # we need this when builing the public getter
-                    node._metadata["func_type"] = ContractFunction.from_AnnAssign(node)
+                    node._metadata["func_type"] = ContractFunction.from_VariableDef(node)
```

Comment:
`from_VariableDef` could probably be renamed to something like `getter_from_VariableDef`